src/utils.ts

```diff
@@ -20,24 +20,24 @@ const stringifyObject = (object) => JSON.stringify(object, null, 2);
 
 /* eslint-disable no-console */
 export const Logger = {
-  log(message) {
-    console.log(chalk.white(message));
+  log(message: string) {
+    console.log(chalk.white(message.replace(/\n|\r/g, '')));
```

Comment:
maybe you could use a helper like this to avoid repetition and better maintainability: 
`const sanitizeMsg = (message: string): string => message.replace(/\n|\r/g, '');`
[...]
` console.log(chalk.white(sanitizeMsg(message)));`